Tests/DependencyInjection/Fixtures/config/yml/dbal_allow_url_override.yml

```diff
@@ -0,0 +1,3 @@
+doctrine:
+    dbal:
+        override_url: true
```

Comment:
This config is not very useful. At least also add `url` and few other options. You would perhaps find that enabling this option overrides host with `localhost` without specifying `host` option

Reply:
> You would perhaps find that enabling this option overrides host with `localhost` without specifying `host` option

Yes I just discovered that earlier today and working up a fix now for it.